src/lib/requestLogger.js

```diff
@@ -81,12 +86,38 @@ const logMessage = (verbosity, message, additionalData = null, notification = tr
       Logger.error(message)
       break
     case 'info':
-    default:
+    default: {
       Logger.info(message)
```

Comment:
We should pass additionalData if that exists for any verbosity

Reply:
ok, I will add it